ngi_reports/common/ign_sample_report.py

```diff
@@ -216,6 +217,34 @@ def parse_snpeff(self):
 
 
 
+    def parse_picard_metrics(self):
+        """ Parse the picard metrics file to get the duplication rates
+        """
+        for sample_id in self.samples.iterkeys():
+
+            # Build the expected filenames
+            snpEff_csv = os.path.realpath(os.path.join(self.working_dir,
```

Comment:
Hah, good spot! Lazy copy and paste error. Apparently I did it twice (see L232) which is why it worked when I tested it :grin: 


Will push a new commit now...